client/src/components/Shared/Icon/Icon.jsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Icon = ({ name, color, width, height, id }) => {
+	switch (name) {
+		case 'logo':
+			return (
+				<svg viewBox="26 70 274 226" className="icon__logo">
```

Comment:
Maybe move these icons outside of the file in the separate one, and then just return correct svg for each case. It's just another level of abstraction but it would be a bit cleaner IMO.